src/screens/FeedScreen/elements/Search.js

```diff
@@ -52,7 +52,7 @@ export const HeaderStyles = StyleSheet.create({
   },
   wrapperSearch: {
     flex: 1,
-    backgroundColor: '#F5F5F5',
+    backgroundColor: COLORS.lightgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded color value to a reference from the `COLORS` object is a good practice for maintainability and consistency. However, ensure that `COLORS.lightgrey` is defined and has the same color value as `'#F5F5F5'`.
```diff
-    backgroundColor: '#F5F5F5',
+    backgroundColor: COLORS.lightgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->